sites/website/package.json

```diff
@@ -4,15 +4,17 @@
   "private": true,
   "scripts": {
     "clean": "node ../../build/clean.js docs",
-    "prepare-docs": "npm run clean && node src/prepare-docs.js",
-    "dev": "npm run prepare-docs && docusaurus start",
-    "build": "npm run prepare-docs && docusaurus build"
+    "prepare-docs": "yarn clean && node src/prepare-docs.js",
+    "build-homepage": "node src/build-homepage.js",
+    "dev": "yarn build-homepage && yarn prepare-docs && docusaurus start",
+    "build": "yarn build-homepage && yarn prepare-docs && docusaurus build"
   },
   "dependencies": {
-    "@docusaurus/core": "2.0.0-alpha.48",
-    "@docusaurus/preset-classic": "2.0.0-alpha.48",
-    "@docusaurus/theme-live-codeblock": "2.0.0-alpha.39",
+    "@docusaurus/core": "2.0.0-alpha.56",
+    "@docusaurus/preset-classic": "2.0.0-alpha.56",
+    "@docusaurus/theme-live-codeblock": "2.0.0-alpha.56",
```

Comment:
This should still be at 39: https://www.npmjs.com/package/@docusaurus/theme-live-codeblock